_pages/docs/testing-with-sjs.md

```diff
@@ -60,7 +60,31 @@ Note that the last line returns the `assertions` array.
 
 ### Testing For Expected Errors
```

Comment:
I don't think these examples are quite what we want here.  Unfortunately we can't use the existing `assertThrowsError()` or `assertThrowsErrorWithMessage()` assertion functions because they are implemented in XQuery and we cannot call a JavaScript function from the XQuery library today.

Instead of showing an example that asserts against our internal failure model, it might make sense to show a simplified JavaScript error being checked, an a failure if the message is not as expected.


For the time being the following example seems more appropriate:
```
try {
    throw "wrong error message";
} catch (error) {
    test.assertEqual("expected error message", error, "Did not find expected error message");
}
```